locales/Ar_ar.json

```
@@ -0,0 +1,139 @@
+		"text": "إن بريتندو بديل مجاني و مصدر مفتوح لسيرفرات ال3دي اس و الوي يو ,يسمح بالاتصال عبر الإنترنت حتي بعد إغلاق السيرفرات الأصلية",
+		"buttons": {
+			"readMore": "أقرأ اكثر"
+		}
+	},
+	"aboutUs": {
+		"title": "معلومات عنا",
+		"paragraphs": [
+			"إن بريتندو مشروع مصدر مفتوح هدفه إعادة إنشاء شبكة نينتندو لل3دي اس و الوي يو عن طريق الهندسة العكسية",
+			".خدماتنا ستكون مجاناً ومصدرها مفتوح فستعيش وقتا طويلاً حتي بعد إغلاق شبكة نينتندو"
+		]
+	},
+	"progress": {
+		"title": "تقدم",
+		"paragraphs": [
+			"نحن نعمل الآن على الميفيرز و الأكونتات وإتصالها مع السيرفرات",
+			"و نعمل علي ماريوكارت 7 لل3دي اس و نريد ان نعمل علي الألعاب الآخري في اسرع وقت"
+		]
+	},
+	"faq": {
+		"title": "بعض الأسئلة المكررة",
+		"text": ":هنا بعض الأسئلة المتكررة بإجابتها",
+		"QAs": [
+			{
+				"question": "ما هو بريتندو؟",
+				"answer": "إن بريتندو مصدر مفتوح بديل عن شبكة نينتندو هدفها إنشاء سيرفرات لعائلة ال3 دي اس و الوي يو هدفنا الحفاظ علي الاجزاء الونلاين و الاستمتاع باقصي حد"
+			},
+			{
+				"question": "هل هويتي الشخصية علي شبكة نينتندو ستستمر علي بريتندو؟",
+				"answer": "للأسف لا اي الهويات الشخصية لشبكة نينتندو لن تعمل علي بريتندو لأن نينتندو لديم المعلومات الشخصية؛ و من الرغم من أن هجرة الهوية الشخصية من شبكة نينتندو إلي بريتندو ممكنة إلا انها لها خطر و لا نريد أن يكون معنا المعلومات  الشخصية"
+			},
+			{
+				"question": "كيف استخدم بريتندو؟",
+				"answer": "إن بريتندو ليس جاهز للاستخدام العام ولكن عندما يكون جاهزا ستسطيع أن نسخدمه من خلال الهومبرو باتشر"
+			},
+			{
+				"question": "هل تعرف متي تكون خدمة او خاصية جاهزة؟",
+				"answer": "لا معظم خدمات بريتندو يتم إنشائها مفردة إي شخص قد يعمل علي الميفيرز و شخصاً أخر قد يعمل علي الاصدقاء و الاكونتات فلا نستطيع أن نعرف متي ستكون جاهزة"
+			},
+			{
+				"question": "هل بريتندو يعمل علي سيمو و المحاكيات؟",
+				"answer": "إن بريتندو مصمم لل3 دي اس و الوي يو؛ و في الوقت الحالي سيمو هو المحاكية الوحيد يدعِم شبكة نينتندو سي مد لا يدعم شبكة بريتندو مباشرا و لكن  ستصتيع استخدامه (حالياً  سيمو لا يدعم شبكة بريتندو)إن بريتندو مصمم لل3 دي اس و الوي يو؛ و في الوقت الحالي سيمو هو المحاكية الوحيد يدعِم شبكة نينتندو سي مد لا يدعم شبكة بريتندو مباشرا و لكن  ستصتيع استخدامه (حالياً  سيمو لا يدعم شبكة بريتندو)"
+			},
+			{
+				"question": "لو اتمنعنت من شبكة بريتندو هل سأستمتر ممتنع من بريتندو؟",
+				"answer": "لا جميع المستخدمين لن يكونوا محرمين و لكن يجب أن تمشي وراء القوانين او سيتمامتناعك"
+			},
+			{
+				"question": "هل سيدعم بريتندو الوى و السوتش؟",
+				"answer": "<a href=\"https://wiimmfi.de/\" target=\"_blank\">الوي لديها سيرفرات مخصصة من ويميفاي</a>   و ليس لدينا الهدف للسويتش بسبب اختلافها عن شبكة نينتندو"
+			},
+			{
+				"question": "هل احتاج أن اهكر حتي اتصل؟",
+				"answer": "نعم ستحتاج إلي تهكير جهازك لكن علي الوي يو تحتاج فقط إلي الهومبرو لانشر اي هاكسي او موخا اما علي ال3 دي اس سيتم الاعلان في وقتا لاحقاً"
+			}
+		]
+	},
+	"credits": {
+		"title": "الفريق",
+		"text": "قابل الفريق وراء المشروع",
+		"people": [
+			{
+				"name": "Jonathan Barrow (RedDucks)",
```

only needs to change the username in between brackets
```suggestion
				"name": "Jonathan Barrow (jonbarrow)",
```